agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -64,3 +64,60 @@ class TemplateDB(Model):
 
     class Config:
         collection = "templates"
+
+
+class EvaluationTypeSettings(EmbeddedModel):
+    similarity_threshold: Optional[float]
+
+
+class EvaluationScenarioInput(EmbeddedModel):
+    input_name: str
+    input_value: str
+
+
+class EvaluationScenarioOutput(EmbeddedModel):
+    variant_name: str
+    variant_output: str
+
+
+class EvaluationDB(Model):
+    status: str
+    evaluation_type: str
+    evaluation_type_settings: EvaluationTypeSettings
+    llm_app_prompt_template: str
+    variants: List[str]
+    app_name: str
+    testset: Dict[str, str]
+    user: UserDB = Reference(key_name="user")
+    created_at: Optional[datetime] = Field(default=datetime.utcnow())
+    updated_at: Optional[datetime] = Field(default=datetime.utcnow())
+
+    class Config:
+        collection = "evaluations"
+
+
+class EvaluationScenarioDB(Model):
```

Comment:
I think in the future we should break this into different types. Right now we have vote, score, and evaluation in the same model.